src/generic/LaurentMPoly.jl

```diff
@@ -18,9 +18,9 @@ elem_type(::Type{LaurentMPolyWrapRing{T, PR}}) where {T, PR} =
 parent(p::LaurentMPolyWrap) = p.parent
 
 base_ring_type(::Type{<:LaurentMPolyWrapRing{T}}) where {T} = parent_type(T)
-
 base_ring(R::LaurentMPolyWrapRing) = base_ring(R.mpolyring)::base_ring_type(R)
 
+coefficient_ring_type(T::Type{<:LaurentMPolyWrapRing}) = base_ring_type(T)
```

Comment:
since `coefficient_ring` does not delegate to `base_ring` of the same argument, I find it unintuitive (or even incorrect, don't know enough about these types) to do that for `coefficient_ring_type`.